src/segments/python.go

```diff
@@ -6,6 +6,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"golang.org/x/exp/slices"
```

Comment:
```suggestion
	"slices"
```

as of 1.21 slices is now part of go itself